Refactor: Basic Components를 제외한 나머지 Components에 theme 적용

lkkikkil opened the pull request:
common 컴포넌트와 header, main 관련 컴포넌트에 적용

lkkikkil committed:
Refactoy: common 컴포넌트 theme 적용

common 컴포넌트에 사용되는 color와 backgroundColor, fontSize에 theme로 적용( border는 보류)

lkkikkil committed:
Refactor: header와 main 관련 멐포넌트들에 theme 적용

HeaderAlarm과 HeaderComponent. PostDetail에서 theme 적용되어 있지 않는
부분들을 적용

lkkikkil committed:
Feat: 헤터 관련 커포넌트 state를 대신할 임시 변수 테스트

state를 이용하기 전 임시로 사용할 변수 만들어 적용

lkkikkil committed:
Feat: PostDetail 컴포넌트 state를 대신할 임시 변수 테스트

state를 이용하기 전  임시로 사용할 변수 만들어 적용

lkkikkil committed:
Fix: 이미지 경로 문제 수정

이미지 경로에 .이 붙어 출력되지 않는 문제 해결

lkkikkil committed:
Feat: PostDetail에서 링크로  이용되는 PostDetailLink 컴포넌트 제작

링크 이름과 링크 URL을 받고 클릭시 URL로 이동하는 기능 구현

lkkikkil committed:
Feat: PostDetail에 PostDetailLink 컴포넌트 적용

state를 사용하기 전 기능 임시로 적용

lkkikkil committed:
Style: profile_img가 undefined 일 때 설정

profile_img가 undefined이면 user.svg 이미지로 기본 설정

Yejin0O0 reviewed (changes requested):
[네이밍]
PostDetailComponent 로 이름 변경해 주세요

lkkikkil committed:
Feat: 헤더 관련 컴포넌트 state에 따른 분리 및 state 적용

헤더 search부분과 오른쪽 기능 부분 컴포넌트 분리 및 state에 따른 요소
보여지는 기능 구현

lkkikkil committed:
Remove: 잘못 만들어진 main 폴더 삭제

main 폴더 삭제

lkkikkil committed:
Feat: 포스트 관련 컴포넌트 폴더 위치 수정 및state 적용

포스트 묶어주는 PostComponent 생성 및 PostDetailLink 컴포넌트 생성
포스트 관련 컴포넌트 state에 따른 요소 보여지는 기능 구현

lkkikkil committed:
Feat: Route별 보여질 페이지 구현

Post 관련 페이지 모두 구현

lkkikkil committed:
Feat: PostItem의 navigate 설정

클릭시 post_idx에 맞는 위치로 경로 설정해주는 기능 구현

lkkikkil committed:
Rename: 헤더 파일에  container 컴포넌트 component로 수정

헤더 파일이름 전부 수정

lkkikkil committed:
Style: vscode 프리티어 적용

모든 코드에 프리티어 적용

lkkikkil committed:
Feat: PostDetailNav 컴포넌트 state 적용

PostDetailNav에 구성요소 state로 가져오도록 적용

lkkikkil committed:
Fix: PostDetailNav 임시 데이터 개수 조정

임시 데이터 60개에서 5개로 조정
